tensorflow/core/kernels/mlir_generated/build_defs.bzl

```diff
@@ -162,6 +162,7 @@ def _gen_kernel_bin_impl(ctx):
             "--output=%s" % gpu_bin.path,
             "--enable_ftz=%s" % (ctx.attr.data_type == "f32"),
             "--cpu_codegen=%s" % ctx.attr.cpu_codegen,
+            "--index_64bit=%s" % ctx.attr.index_64bit,
```

Comment:
Here, we do not want to enforce i64 indexing for everything but only for large tensors. The right option to use here would be to use `jit-i64-indexed-for-large-tensors`, which may need to be added to the kernel generator CLI.

Reply:
The argument is changed and also added to CLI